packages/components/link/src/Link.tsx

```diff
@@ -0,0 +1,24 @@
+import type { LinkProps } from './link.types';
+import { LinkStyled } from './link.styles';
+import { useTheme } from '@douro-ui/react';
+import { FC } from 'react';
+
+const Link: FC<LinkProps> = ({ underline, fontSize, ...props }) => {
+  const theme = useTheme();
+
+  return (
+    <LinkStyled
```

Comment:
A link typically has a href, target, and possibility accessibility attributes 'Aria'. 
Please read a little bit of this article: https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/

Reply:
Ignore this comments. Passing `...props` is enough.